cnc_ctrl_v1/Report.cpp

```diff
@@ -199,6 +200,7 @@ void reportMaslowSettings() {
     Serial.print(F(" (z axis Kp Velocity)\r\n$34=")); Serial.print(sysSettings.zKiV);
     Serial.print(F(" (z axis Ki Velocity)\r\n$35=")); Serial.print(sysSettings.zKdV);
     Serial.print(F(" (z axis Kd Velocity)\r\n$36=")); Serial.print(sysSettings.zPropWeightV);
+    Serial.print(F(" (chain sag correction value, $K)\r\n$37=")); Serial.print(sysSettings.chainSagCorrection);
```

Comment:
There is a mistake here.  It is confusing, but the start of the line describes the prior setting.  So what you want is the following line " (zaxis Velocity..." to be the start of this line and the " (chain sag correction ..." to be the following line.

Reply:
Thank you! I was hoping I would be able to borrow your eyes to validate my code. I'll fix this.